### ISSUE TITLE
NetBox 3.2 Support

### ISSUE THREAD

**Issue by ryanmerolle** [devon-mar/netbox-lists#1]
[NetBox 3.2.0](https://github.com/netbox-community/netbox/releases/tag/v3.2.0) was just released and has a number of [extension improvements](https://github.com/netbox-community/netbox/issues/8333).

Furthermore, a [NetBox 3.2 plugin tutorial](https://github.com/netbox-community/netbox-plugin-tutorial) was released

**Comment by devon-mar** [comment]
@ryanmerolle I've already been running CI on the ``devel`` branch against the 3.2 beta and have just switched it to the final release. 

As for the plugin improvements, I don't think that there are any that this plugin can take advantage of.

**Comment by ryanmerolle** [comment]
This was just a courtesy issue I have raised with most plugins in to notify maintainers of the changes.  Feel free to close this issue if you like.

**Comment by dmcken** [comment]
At the moment netbox (I run it under docker) fails to start when upgraded to 3.2:


```
Traceback (most recent call last):
  File "/opt/netbox/netbox/./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 414, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 455, in execute
    self.check()
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 487, in check
    all_issues = checks.run_checks(
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/urls/resolvers.py", line 480, in check
    for pattern in self.url_patterns:
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/urls/resolvers.py", line 696, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/urls/resolvers.py", line 689, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/netbox/netbox/netbox/urls.py", line 9, in <module>
    from extras.plugins.urls import plugin_admin_patterns, plugin_patterns, plugin_api_patterns
  File "/opt/netbox/netbox/extras/plugins/urls.py", line 35, in <module>
    urlpatterns = import_object(f"{plugin_path}.api.urls.urlpatterns")
  File "/opt/netbox/netbox/extras/plugins/utils.py", line 31, in import_object
    spec.loader.exec_module(module)
  File "/opt/netbox/venv/lib/python3.9/site-packages/netbox_lists/api/urls.py", line 3, in <module>
    from .views import (
  File "/opt/netbox/venv/lib/python3.9/site-packages/netbox_lists/api/views.py", line 27, in <module>
    from .renderers import PlainTextRenderer
  File "/opt/netbox/venv/lib/python3.9/site-packages/netbox_lists/api/renderers.py", line 1, in <module>
    from django.utils.encoding import smart_text
ImportError: cannot import name 'smart_text' from 'django.utils.encoding' (/opt/netbox/venv/lib/python3.9/site-packages/django/utils/encoding.py)

```

**Comment by devon-mar** [comment]
Hi @dmcken,

Yes, the current package on pypi hasn't been updated yet to support NetBox 3.2. I'll try to get the next version out within the week.

**Comment by devon-mar** [comment]
@dmcken I have published 2.0 and (2.1.1 now) which have support for NetBox 3.2.
